weblab/entities/models.py

```diff
@@ -61,7 +62,7 @@ def add_file_to_repo(self, file_path):
         self.repo.index.add([file_path])
 
     def commit_repo(self, message, author_name, author_email):
```

Comment:
I'd really like some docstrings here! What does this method return? What happens with git or connection errors?

Reply:
I'm adding some more docstrings to these repository methods. Will aim to keep docstring coverage more up to date in future commits...